dev/drivers/scripts/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.sh

```diff
@@ -9,46 +9,31 @@
 #PBS -l debug=true
 #PBS -V
 
+set -x
+
 #%include <head.h>
 #%include <envir-p1.h>
 
-############################################################
-# Load modules
-############################################################
-module reset
-export HOMEevs=/lfs/h2/emc/vpppg/save/$USER/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/Fork/EVS
 source ${HOMEevs}/versions/run.ver
 
-module use /apps/ops/para/libs/modulefiles/compiler/intel/${intel_ver}
-export HPC_OPT=/apps/ops/para/libs
-module use /apps/dev/modulefiles/
-module load ve/evs/${ve_evs_ver}
-module load gsl/${gsl_ver}
-module load netcdf/${netcdf_ver}
-module load met/${met_ver}
-module load metplus/${metplus_ver}
-
-module load libjpeg/$libjpeg_ver
-module load grib_util/$grib_util_ver
-module load prod_util/${produtil_ver}
-module load prod_envir/${prodenvir_ver}
-
-module load geos/${geos_ver}
-module load proj/${proj_ver}
-module load imagemagick/${imagemagick_ver}
-
-module list
-
 export NET=evs
 export COMPONENT=hurricane
 export RUN=global_det
 export STEP=plots
 export VERIF_CASE=tcgen
-
 export envir=dev
 export cyc=00
 export job=jevs_hurricane_global_det_tcgen_plots_${cyc}
 
+############################################################
+# Load modules
+############################################################
+module reset
+source ${HOMEevs}/modulefiles/${COMPONENT}/${COMPONENT}_${STEP}.sh
+module load geos/${geos_ver}
```

Comment:
@OliviaOstwald-NOAA , let's get lines 34 and 35 out of the driver script and into the modules file.
`module load geos/${geos_ver}`
`module load proj/${proj_ver}`

Reply:
done!